common/utils/hwctrace.h

```diff
@@ -17,6 +17,11 @@
 #ifndef COMMON_UTILS_HWCTRACE_H_
 #define COMMON_UTILS_HWCTRACE_H_
 
+#ifdef ANDROID
+#define ATRACE_TAG ATRACE_TAG_GRAPHICS
```

Comment:
can we move this to https://github.com/01org/IA-Hardware-Composer/blob/master/os/android/platformdefines.h